CRM/Event/Form/SelfSvcUpdate.php

```diff
@@ -95,6 +95,7 @@ class CRM_Event_Form_SelfSvcUpdate extends CRM_Core_Form {
    * @var bool
    */
   protected $isBackoffice = FALSE;
+  private string $_userContext;
```

Comment:
@jofranz I don't think we can add type hints here yet for 7.4 support (note there is a style fail too)

Reply:
That's correct.
https://github.com/civicrm/civicrm-core/blob/856b667d318b79b92d37641162a67ebc5706ae5c/CRM/Upgrade/Incremental/General.php#L38-L43